core/domain/email_manager.py

```diff
@@ -631,11 +705,18 @@ def get_moderator_unpublish_exploration_email():
     except utils.ValidationError:
         return ''
 
-    return config_domain.Registry.get_config_property(
-        'unpublish_exploration_email_html_body').value
+    config_prop = config_domain.Registry.get_config_property(
+        'unpublish_exploration_email_html_body'
+    )
+    # Ruling out the possibility of None for mypy type checking.
+    assert config_prop is not None
+    config_prop_value = config_prop.value
+    # Ruling out the possibility of Any for mypy type checking.
+    assert isinstance(config_prop_value, str)
+    return config_prop_value
```

Comment:
Hey, can you create an issue for me to investigate if this can be simplified? I think it might be possible to create some generics for this.

Reply:
I have created an issue #15702  . PTAL!

Reply:
Thanks!